core/src/dird/ua_output.cc

```diff
@@ -1224,7 +1224,7 @@ static bool ParseListBackupsCmd(UaContext* ua,
     return false;
   }
 
-  selection.bsprintf("AND Job.Type='B' AND Client.Name='%s' ",
+  selection.bsprintf("AND Job.Type IN('B','a') AND Client.Name='%s' ",
```

Comment:
```suggestion
  selection.bsprintf("AND Job.Type IN('B','A') AND Client.Name='%s' ",
```
According to include/job_types.h JT_ARCHIVE is 'A' not 'a' for Archive Jobs.